src/modules/roc_fec/target_openfec/roc_fec/of_decoder.cpp

```diff
@@ -294,11 +294,11 @@ void OFDecoder::reset_session_() {
 
     if (OF_STATUS_OK
         != of_set_callback_functions(
-            of_sess_, source_cb_,
-            // OpenFEC doesn't repair fec-packets in case of Reed-Solomon FEC
-            // and prints curses to the console if we give it the callback for that
-            codec_id_ == OF_CODEC_REED_SOLOMON_GF_2_M_STABLE ? NULL : repair_cb_,
-            (void*)this)) {
+               of_sess_, source_cb_,
```

Comment:
Could you please rebase on the develop branch? It seems that we fixed this formatting recently in https://github.com/roc-project/roc/commit/fa43b484a36da280e0aa7b9b787f7d2010be4421.